app/backend/src/couchers/db.py

```diff
@@ -118,19 +118,32 @@ def new_password_reset_token(session, user, hours=2):
     return password_reset_token, f"{hours} hours"
 
 
-def set_email_change_token(session, user, hours=2):
+def set_email_change_tokens(session, user, double_confirmation, hours=2):
```

Comment:
I think `confirm_old_email` is a tad more descriptive than `double_confirmation`?